tests/e2e/maps/apis.ts

```diff
@@ -7,4 +7,10 @@ export class ApisWidget {
         await this.page.waitForSelector("api-list.block");
         await this.page.waitForSelector("api-list div.table div.table-body div.table-row");
     }
+
+    public async getApissCount(): Promise<number | undefined> {
```

Comment:
I am ssssnake :D

Please remove one s ;)

Reply:
updated, thank you for noticing